test/test/lock/LockTests.java

```diff
@@ -48,4 +19,16 @@ public void datagramSocketLock(TestProcess p) throws Exception {
         out = p.profile("-e lock -d 3 -o collapsed");
         assert out.samples("sun/nio/ch/DatagramChannelImpl.send") > 10;
     }
+
+    @Test(mainClass = RaceToLock.class, inputs = { "0" }, args = "1000", output = true)
+    @Test(mainClass = RaceToLock.class, inputs = { "10000" }, args = "1000", output = true)
+    @Test(mainClass = RaceToLock.class, inputs = { "1000000000" }, args = "1000", output = true)
+    public void raceToLocks(TestProcess p) throws Exception {
+        int interval = Integer.parseInt(p.inputs()[0]);
+
+        p.profile("--lock " + interval + " --threads -o collapsed");
```

Comment:
The test does not check profiling results at all, is it intentional?

Reply:
Yes, currently it is only checking the wait times in stdout of the test app.